redis/db/redisbp/example_hooks_test.go

```diff
@@ -6,7 +6,9 @@ import (
 	"github.com/go-redis/redis/v8"
 	"github.com/opentracing/opentracing-go"
 
+	"github.com/reddit/baseplate.go/internal/prometheusbp"
```

Comment:
also this is an example so services actually don't have access to those `internal` packages 😅

Reply:
right but then what are they supposed to do ? Not instrument ?

Reply:
they can instrument other things they want to instrument (for example, we can define a customized prometheus metric in this example and instrument that as the example)

Reply:
oh yeah I think I misread this file. I'll remove changes completely from here.